tests/test_encode_decode.py

```diff
@@ -16,7 +17,19 @@ class TestBinASCII(unittest.TestCase):
 
 class TestColorsys(unittest.TestCase):
     # TODO: https://docs.python.org/3/library/colorsys.html
-    pass
+    @given(
+        r=st.floats(0.0, 1.0),
+        g=st.floats(0.0, 1.0),
+        b=st.floats(0.0, 1.0),
+        h=st.floats(0.0, 1.0),
+        s=st.floats(0.0, 1.0),
+        v=st.floats(0.0, 1.0),
+    )
+    def test_rgb_hsv_round_trip(self, r, g, b, h, s, v):
+        hsv = colorsys.rgb_to_hsv(r, g, b)
+        rgb = colorsys.hsv_to_rgb(hsv[0], hsv[1], hsv[2])
+
+        self.assertTupleEqual((r, g, b), rgb)
```

Comment:
Normally you don't need to specify `assertXEqual`, `assertEqual` will handle that for you.

In this case, though, you need to call `assertAlmostEqual` on each of the elements, because these are floats, so something like:

```python
import itertools
...
    def test_rgb_hsv_round_trip(self, r, g, b):
        ...

        for expected, actual in itertools.zip_longest((r, g, b), rgb)):
            self.assertAlmostEqual(expected, actual)
```

If you're planning on doing a lot of these, it might be worth abstracting it into `assertSequenceAlmostEqual` or something of that nature.